### PR TITLE
Data quality changes

### PR DESCRIPTION
**Pull request by NitishAroraKPMG**

### CODE REVIEW

**Review by SamerIssam — APPROVED** [review]

File: transit_odp/data_quality/tables/base.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
     ServicePatternStop,
     VehicleJourney,
```
Comment (pritish-ranjan01) [inline transit_odp/data_quality/tables/base.py:10]: Need to discuss

File: transit_odp/data_quality/tables/base.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
...
 )
+from waffle import flag_is_active
```
Comment (pritish-ranjan01) [inline transit_odp/data_quality/tables/base.py:12]: Need to discuss

File: transit_odp/data_quality/tables/base.py [hunk 3]
```diff
@@ -234,3 +237,25 @@ def get_date_string_or_unknown(self, vehicle_journey):
             date = "unknown date"
         finally:
             return date
+
+
+class DQSWarningListBaseTable(GovUkTable):
```
Comment (pritish-ranjan01) [inline transit_odp/data_quality/tables/base.py:242]: Can we move this to dqs app?

File: transit_odp/data_quality/templates/data_quality/snippets/dqs_custom_table.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
+{% load django_tables2 %}
```
Comment (pritish-ranjan01) [inline transit_odp/data_quality/templates/data_quality/snippets/dqs_custom_table.html:1]: Can we move this to dqs app?

File: transit_odp/data_quality/templates/data_quality/snippets/dqs_custom_table.html [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        {% block table.tbody %}
+          <tbody class="govuk-table__body {{ table.attrs.tbody.class |default_if_none:"" }}">
+          {% for row in table.paginated_rows %}
+            {% block table.tbody.row %}    
+              <tr class="govuk-table__row {{ row.attrs.class }}">                            
+                <td class="govuk-table__cell {{ column.attrs.td.class |default_if_none:"" }}">
+                  {{row.cells.line_name}} - {{row.cells.service_code}}
+                  
+                    <!-- <a href="#">
+                      "{% url 'line-detail' pk1=request.resolver_match.kwargs.pk1 pk=request.resolver_match.kwargs.pk host hosts.publish %}?line={{row.cells.line_name}}&service={{row.cells.service_code}}"
```
Comment (pritish-ranjan01) [inline transit_odp/data_quality/templates/data_quality/snippets/dqs_custom_table.html:35]: Can we remove this?

File: transit_odp/data_quality/templates/data_quality/snippets/dqs_custom_table.html [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+                      </a>
+                    -->
+                    
+                  
+                  </td>
+                  <td class="govuk-table__cell {{ column.attrs.td.class |default_if_none:"" }}">
+                    {% if table.attrs.tbody.is_details_link %}
+                      
+                        <!-- TODO: We need to modify the link /detail?line=&service_code=
+                            <a href="#">
```
Comment (pritish-ranjan01) [inline transit_odp/data_quality/templates/data_quality/snippets/dqs_custom_table.html:45]: Can we remove this?

**Pull request merged**